cfme/tests/optimize/test_bottlenecks.py

```diff
@@ -107,4 +109,53 @@ def test_bottlenecks_time_zome(temp_appliance_extended_db, db_restore, db_tbl, d
         # Changing time zone
         view.report.time_zone.fill('(GMT-04:00) La Paz')
         row = view.report.event_details[0]
-        assert row[0].text == (db_row[0][0] - timedelta(hours=4)).strftime("%m/%d/%y %H:%M:%S BOT")
+        assert row[0].text == (db_row[0][0] - timedelta(hours=4)).strftime("%m/%d/%y %H:%M:%S -04")
+
+
+@pytest.mark.tier(2)
+def test_bottlenecks_summary_event_groups(temp_appliance_extended_db, db_restore, db_tbl,
+                                          db_events):
+    """ Checks event_groups selectbox in summary tab. It should filter events by type """
+    with temp_appliance_extended_db:
+        view = navigate_to(Bottlenecks, 'All')
+        # Enabling this option to show all possible values
+        view.summary.show_host_events.fill(True)
+        view.summary.event_groups.fill('Capacity')
+        events = view.summary.chart.get_events()
+        # Compare number of events in chart with number of rows in db
+        assert len(events) == db_events.filter(db_tbl.event_type == 'DiskUsage').count()
+        view.summary.event_groups.fill('Utilization')
+        events = view.summary.chart.get_events()
+        assert len(events) == db_events.filter(db_tbl.event_type != 'DiskUsage').count()
+
+
+@pytest.mark.tier(2)
+def test_bottlenecks_summary_show_host_events(temp_appliance_extended_db, db_restore, db_events):
+    """ Checks host_events checkbox in summary tab. It should show or not host events """
+    with temp_appliance_extended_db:
+        view = navigate_to(Bottlenecks, 'All')
+        view.summary.show_host_events.fill(False)
+        # Checking that events with value 'Host / Node' absent in table
+        events = view.summary.chart.get_events()
+        assert not sum(1 for event in events if event.type == 'Host')
+        view.summary.show_host_events.fill(True)
+        events = view.summary.chart.get_events()
+        # Compare number of events in chart with number of rows in db
+        assert len(events) == db_events.count()
+
+
+@pytest.mark.tier(2)
+def test_bottlenecks_summary_time_zone(temp_appliance_extended_db, db_restore, db_tbl, db_events):
+    """ Checks time zone selectbox in summary tab. It should change time zone of events in chart """
+    with temp_appliance_extended_db:
+        view = navigate_to(Bottlenecks, 'All')
+        events = view.summary.chart.get_events()
+        # Selecting row by uniq value
+        db_row = db_events.filter(db_tbl.message == events[0].message)
+        # Compare event timestamp with db
+        assert events[0].time_stamp == db_row[0][0].strftime("%Y-%m-%d %H:%M:%S UTC")
```

Comment:
This looks like ISO with UTC: 
https://github.com/ManageIQ/integration_tests/blob/master/utils/timeutil.py#L19

import parsetime and use that instead.  Doesn't really help for the test above unfortunately, where you're working with a different offset not in this util.